uspace/lib/c/arch/mips32/src/syscall.c

```diff
@@ -60,7 +60,12 @@ sysarg_t __syscall(const sysarg_t p1, const sysarg_t p2, const sysarg_t p3,
 	       * We are a function call, although C
 	       * does not know it.
 	       */
-	    : "%ra"
+	    : "%ra",
```

Comment:
To make things consistent, the colon should now (that there are two clobbers) go before the comment.